drivers/flash/nrf_qspi_nor.c

```diff
@@ -33,6 +33,10 @@
 
 LOG_MODULE_REGISTER(qspi_nor, CONFIG_FLASH_LOG_LEVEL);
 
+static const struct flash_parameters qspi_flash_parameters = {
+	.erase_value = 0xff,
```

Comment:
missing `.write_block_size`.  Multiple drivers.

Reply:
Eh. Failed to properly apply patches between branches...